packages/cozy-mespapiers-lib/package.json

```diff
@@ -45,6 +45,7 @@
     "date-fns": "2.23.0",
     "flexsearch": "0.7.31",
     "fuse.js": "6.6.2",
+    "react-markdown": "8.0.7",
```

Comment:
You have to use snarkdown, not react-markdown. Way more lighter

Reply:
snarkdown ne supporte pas le target blank https://github.com/developit/snarkdown/issues/5 , et quitte à devoir le tuner pour faire ça, je pense que je vais plutôt supprimer le markdown et faire à l'ancienne en découpant le texte dans les blocs. L'idée de l'approche markdown était justement d'utiliser quelque chose de plus haut niveau pour gérer les textes, à mon sens autant voir une lib qui fait ça clé en main, ou ne rien mettre du tout...